src/main/java/uk/gov/companieshouse/api/accounts/transaction/Resources.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.api.accounts.transaction;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Date;
+import java.util.Map;
+
+public class Resources {
+
+    @Field("updated_by")
+    @JsonProperty("updated_by")
+    private Map<String, String> updatedBy;
```

Comment:
Why isn't updatedBy a User object? "user" appears multiple times in the spec containing required fields. Wouldn't that be better as a User object?
```
"updated_by": {
   "type": "object",
   "description": "The user who last updated the resource.",
   "readOnly": true,
      "items": {
         "$ref": "http://localhost:3123/swagger-2.0/models/transactions.json#/definitions/user"
     }
},
```

Reply:
Not sure, this was how it was done in abridged. I just copied it across and since it is a temporary class I didn't touch it as it will be removed soon.